migration-roadmap/checkbox.md

```diff
@@ -0,0 +1,284 @@
+| `.spectrum-Checkbox.is-indeterminate`                                                       | `indeterminate` attribute                            | Implemented      |
+| `.spectrum-Checkbox.is-invalid`                                                             | `invalid` attribute                                  | Implemented      |
+| `.spectrum-Checkbox-input`                                                                  | Internal checkbox input element, `#input`            | Implemented      |
+| `.spectrum-Checkbox-input:checked`                                                          | `checked` attribute                                  | Implemented      |
+| `.spectrum-Checkbox-input:disabled`                                                         | `disabled` attribute                                 | Implemented      |
+| `.spectrum-Checkbox-box`                                                                    | Internal checkbox box element, `#box`                | Implemented      |
+| `.spectrum-Checkbox-checkmark`                                                              | Checkmark icon element, `#checkmark`                 | Implemented      |
+| `.spectrum-Checkbox-partialCheckmark`                                                       | Indeterminate dash icon element, `#partialCheckmark` | Implemented      |
+| `.spectrum-Checkbox-label`                                                                  | Default slot content                                 | Implemented      |
+| `.spectrum-Checkbox:lang(ja)`, `.spectrum-Checkbox:lang(ko)`, `.spectrum-Checkbox:lang(zh)` | Language-specific styling                            | Implemented      |
+| `.spectrum-Checkbox.is-readOnly`                                                            | `readonly` attribute                                 | Implemented      |
+|                                                                                             | `name` attribute                                     | Missing from CSS |
+|                                                                                             | `tabindex` attribute                                 | Missing from CSS |
+|                                                                                             | `autofocus` attribute                                | Missing from CSS |
+
+## Summary of changes
+
+### CSS => SWC implementation gaps
+
+**New for S2:**
+The checkbox component in Spectrum 2 has the new down state (active) perspective shift applied. Additionally, the checkbox CSS was expanded to ensure coverage of a variety of interactive and variant states: hover, focus-visible, hover+disabled, checked+disabled, focus-visible+checked+hover, indeterminate+invalid, invalid+focused, disabled+read-only, etc.
```

Comment:
I think the down state was the "biggest" thing that was added to checkbox. If I remember correctly, there's some open questions still about whether or not the invalid checkbox has a red border, or if we rely on negative help text only. 

Anyways, is the down state and expansion of testing worth mentioning at all? The down state is mainly CSS, but we do use JS to calculate those properties for some components. The testing bit...I think I'm on the side of removing it.